screensaver.video/addon.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="screensaver.video" name="VideoScreensaver" version="1.3.0" provider-name="robwebset">
+<addon id="screensaver.video" name="VideoScreensaver" version="1.4.0" provider-name="robwebset">
 	<requires>
 		<import addon="xbmc.python" version="2.14.0"/>
 		<import addon="script.module.simplejson" version="3.3.0"/>
 	</requires>
 	<extension point="xbmc.ui.screensaver" library="default.py"/>
-  	<extension point="xbmc.python.script" library="screensaver.py" />
-	<extension point="xbmc.service" library="service.py" start="login">
-		<provides>executable</provides>
+  	<extension point="xbmc.python.script" library="screensaver.py"/>
+	<extension point="xbmc.service" library="service.py" start="login"/>
+	<extension point="xbmc.python.pluginsource" library="plugin.py">
+		<provides>video</provides>
```

Comment:
I think we'll have to agree to dis-agree on this one. I view the addon as a complete set of features - regarless of the extension points implemented - I think it's valid for me to say "If you want you use VideoScreensaver, you need to take all of it". After all addons are optional - and if used, should be used the way that the author intended :)